source/draw/RasterMonochrome.ooc

```diff
@@ -182,14 +182,19 @@ RasterMonochrome: class extends RasterPacked {
 			result = this distance(converted)
 			converted referenceCount decrease()
 		} else {
+			hasCropBox := !cropBox hasZeroArea
+			startX := hasCropBox ? cropBox left : 0
+			startY := hasCropBox ? cropBox top : 0
+			endX := hasCropBox ? cropBox right : this size x
+			endY := hasCropBox ? cropBox bottom : this size y
 			sizeX := this size x
 			sizeY := this size y
```

Comment:
Well, they're still used, but maybe there's not much gain in putting them in local variables now?